dlib/CMakeLists.txt

```diff
@@ -865,8 +869,10 @@ if (NOT TARGET dlib)
       PUBLIC ${dlib_needed_public_includes}
       PRIVATE ${dlib_needed_private_includes}
       )
-   target_link_libraries(dlib PUBLIC ${dlib_needed_public_libraries})
+   target_link_libraries(dlib PUBLIC ${dlib_needed_public_libraries} ${dlib_needed_public_ldflags})
    target_link_libraries(dlib PRIVATE ${dlib_needed_private_libraries})
+   target_compile_options(dlib PUBLIC ${dlib_needed_public_cflags})
+   # target_link_options(dlib PUBLIC ${dlib_needed_public_ldflags})
```

Comment:
```suggestion
```

Reply:
https://cmake.org/cmake/help/latest/command/target_link_options.html
New in version 3.13